account/views.py

```diff
@@ -146,6 +146,48 @@ def get(self, request):
             return Response({'status': 'success', 'Response': {'email':email, 'first_name':first_name, 'last_name':last_name, 'company':company, 'manager':manager}}, 
                                 status=status.HTTP_200_OK)
 
+        except Exception as e:
+            return Response({'status': False, 'message': str(e)},
+                            status=status.HTTP_400_BAD_REQUEST)
+
+employee_id = openapi.Parameter('id', in_=openapi.IN_QUERY,
```

Comment:
try to include `employee_id` inside the `GetEmployeeInfo`'s APIView if you don't plan to reuse `employee_id` anywhere else. It's just a way to not pollute the global scope with these variables.